etc/dbus-serialbattery/reinstall-local.sh

```diff
@@ -317,23 +317,12 @@ if [ "$bluetooth_length" -gt 0 ]; then
             echo "exec 2>&1"
             echo "echo"
             echo "echo"
-            echo "echo \"INFO:Preparing Bluetooth for connection to BMS\""
-            echo "echo \"INFO:Bluetooth details\""
-            # close all open connections, else the driver can't connect
-            echo "bluetoothctl disconnect $3"
-
-            # enable bluetoothctl scan in background to display signal strength (RSSI), else it's missing
-            echo "bluetoothctl scan on | grep \"$3\" | grep \"RSSI\" &"
-            # with multiple Bluetooth BMS one scan for all should be enough. Check if that can be changed globally, maybe with a cronjob after reboot?
-            # echo "bluetoothctl scan on > /dev/null &"
-
-            # wait 5 seconds to finish the scan
-            echo "sleep 5"
-            # display some Bluetooth device details
-            echo "bluetoothctl info $3 | grep -E \"Device|Alias|Pair|Trusted|Blocked|Connected|RSSI|Power\""
+            echo "echo \"INFO:Starting dbus-serialbattery.py\""
             echo "echo"
+	    # close earlier unclosed conections to the BMS, timeout after 4s
+	    echo "python3 -c \"import os, time; os.popen('bluetoothctl disconnect $3');time.sleep(4)\""
```

Comment:
Is there a reason why you chose this the inefficient way? This shell command calls python that calls a shell command.

Just call
```bash
echo "bluetoothctl disconnect $3; sleep 5"
```
as before.

Reply:
Yes there is a reason, sometimes "bluetoothctl disconnect" freezes and never exits.

This is a way of setting a timeout so that we can ignore the fact that it does not exit.

I remember I found a shell command that could do it, but it was not available in the venusOS image so I just made it in to a python one liner.

The command:
`echo "bluetoothctl disconnect $3; sleep 5"`
Would lead to a behaviour where we disconnect then wait 5 seconds for no reason. Which is different from the intended behaviour.

Maybe we could do:
```
echo "bluetoothctl disconnect $3 &"
echo "sleep 4"
```
But I am not sure if bash waits for the ampersand command to exit before it ends at the final line.

Reply:
sounds like things are in a state for me to try putting this PR onto my install? Can that be done using the install.sh method? Or do I need to manually copy files over?

Reply:
You can try. Copy over the 4 files that have been altered.

There are changes made to the code since I last tested, so I don't know if there are any small typos or things like that.
But the worst that could happen is that you get an error message in the log.

Reply:
Well, that worked surprisingly easy! 

Copied the files over into the right directory, went back to the original install/setup documentation, looked through the config.default.ini, edited the device config.ini with the right info, ran reinstall-local.sh...and it worked! I see battery info, cell voltage, etc in the GUI (v2).

Might not be perfect, might need to tweak some of the cell voltage settings, but here's what I used 

```
[DEFAULT]
MAX_BATTERY_CHARGE_CURRENT    = 100.0
MAX_BATTERY_DISCHARGE_CURRENT = 200.0

BMS_TYPE=LiTime_Ble
BLUETOOTH_BMS=LiTime_Ble <MAC ADDRESS HERE>
MIN_CELL_VOLTAGE   = 3.000
MAX_CELL_VOLTAGE   = 3.450
FLOAT_CELL_VOLTAGE = 3.375
```

Although it does seem to lose some information, now and then, or at least when the UI is refreshed, I need to go into Devices, then select the battery. And then it displays in the Brief. 

And then as I was trying to connect via BT on the Victron App, it seemed to lose track of the battery entirely. Is this because the BT stack gets reset by VenuOS, and thus no longer is monitoring the BT LE data, and so 'forgets' the battery? 

I then rebooted, it was still gone, ended up running restore-gui.sh, and it appears back in the Brief and in the Devices List. 

Browsing around some more, it disappeared (went disconnected as per the GUI?), and then I ended up click the Disable Bluetooth, which seems to just disable the BT for being discovered, e.g. Victron Connect app. 

So seems successful, although not sure if this will be as useful as I really want. I'd much prefer to be able to get all the info, everything, via BT app, or without needing internet connection/VRM to easily connect. While I do have Starlink for the trailer, it's not something that I'll be using constantly, so I'll have it, and the associated WiFi, off unless I'm actually using it to save power.

Well, the good news is I can see the cell voltages this way! 

So not sure if this code is quite ready to be merged or not, but seems to be working fine with some quirks that I think are just due to BT and VenusOS and how things work together (nor not quite).

Reply:
Right you have to follow this:
https://github.com/mr-manuel/venus-os_dbus-serialbattery/blob/0671ae0a0063ac9fb42dd45359923a7410889daa/etc/dbus-serialbattery/reinstall-local.sh#L440